packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -1765,6 +1765,16 @@ export class AutoScalingGroup extends AutoScalingGroupBase implements
       };
     }
   }
+
+
+  private validateTargetGroup(): string[] {
+    const errors = new Array<string>();
+    if (this.hasSetScaleOnRequest && this.targetGroupArns.length > 1) {
+      errors.push('Cannon use multiple target groups if `setScaleOnRequest()` is being used.');
```

Comment:
`setScaleOnRequest()` should be `scaleOnRequestCount()`, no? `scaleOnRequestCount()` enables `hasSetScaleOnRequest()`, I don't see a `setScaleOnRequest()` method in the `auto-scaling-group.ts`

Reply:
Oops, the "set" part in the variable meant to reflect that the method had been called; Seems I transposed that to method name when writing the error message. Changed in 1f3f3f00145308a2e81106297d2277da4b539241